lib/budgie/tracking.ex

```diff
@@ -180,4 +198,27 @@ defmodule Budgie.Tracking do
         query
     end)
   end
+
+  def ensure_join_link(%Budget{} = budget) do
+    %BudgetJoinLink{}
+    |> BudgetJoinLink.changeset(%{budget_id: budget.id})
+    |> Repo.insert(
+      on_conflict: {:replace, [:updated_at]},
```

Comment:
For now, only allow a single join link per budget.
Future plans might include rotation, expiration, and revocation.